src/haxelib/api/Vcs.hx

```diff
@@ -193,6 +194,18 @@ abstract class Vcs implements IVcs {
 		// just in case process hangs waiting for stdin
 		p.stdin.close();
 
+		// In certain cases of git clones, it will hang on reading from stderr
+		// if we don't read from it. So we will always try to read from it.
```

Comment:
The reason this was solved with threads on windows is that the same thing could happen with p.stdout and p.stderr at the same time, in which case the only solution is to read from both simultaneously (with separate threads).

However, the threads caused issues on Linux so I made it windows only in: b8c24a783000c609ae3b45c27aab423d9e0d7193. I suspect the same issue is also behind these crashes: #591